flexmeasures/api/v1_3/__init__.py

```diff
@@ -1,16 +1,24 @@
 from flask import Flask, Blueprint
 
-from flexmeasures.api.common.utils.deprecation_utils import deprecate_blueprint
+from flexmeasures.api.common.utils.deprecation_utils import (
+    deprecate_blueprint,
+    sunset_blueprint,
+)
 
 # The api blueprint. It is registered with the Flask app (see app.py)
 flexmeasures_api = Blueprint("flexmeasures_api_v1_3", __name__)
 deprecate_blueprint(
     flexmeasures_api,
     deprecation_date="2022-12-14",
     deprecation_link="https://flexmeasures.readthedocs.io/en/latest/api/v1_3.html",
-    sunset_date="2023-02-01",
+    sunset_date="2023-05-01",
     sunset_link="https://flexmeasures.readthedocs.io/en/latest/api/v1_3.html",
 )
+sunset_blueprint(
+    flexmeasures_api,
+    "1.3",
+    "https://flexmeasures.readthedocs.io/en/latest/api/v1_3.html",
```

Comment:
Maybe https://flexmeasures.readthedocs.io/en/v0.12.3/api/v1_3.html would be a more stable option here (and in the other versions).

Reply:
Good point. I'll point to the 0.13 snapshot instead, which will have the sunset notice *and* the link to the previous version with the full documentation for API v1.3.